internal/bminventory/inventory.go

```diff
@@ -1878,10 +1878,10 @@ func (b *bareMetalInventory) PostStepReply(ctx context.Context, params installer
 		handlingError := b.handleReplyError(params, ctx, &host)
 		if handlingError != nil {
 			log.WithError(handlingError).Errorf("Failed handling reply error for host <%s> cluster <%s>", params.HostID, params.ClusterID)
+			return installer.NewPostStepReplyInternalServerError().
```

Comment:
This error contains all the text that the agent already sent to the assisted-service (which might be long).  There is no reason to send this whole text back to the agent for just logging.  It was already logged there.

Reply:
@ori-amizur this log will contain only the handling error - it will not include the agent error. For example: if there was an error for changing the host status to error - we will write it in the log.

Reply:
@ori-amizur I fixed it and now I return 200